discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -213,6 +213,21 @@ def wait_for_one_host_to_be_in_wrong_boot_order(self, fall_on_error_status=True)
             fall_on_error_status=fall_on_error_status,
         )
 
+    def wait_for_hosts_to_be_in_wrong_boot_order(
+            self,
+            nodes_count=env_variables['num_nodes'],
+            timeout=consts.CLUSTER_INSTALLATION_TIMEOUT,
+            fall_on_error_status=True
+    ):
+        utils.wait_till_all_hosts_are_in_status(
```

Comment:
rename to wait_till_hosts_are_in_status

Reply:
this function was here before, not that I have a problem to rename it but this isn't a new function and will include refactoring out of the task scope.

Reply:
OK so leave it as is.